manifests/stage-rdma-device-plugin/0020-k8s-rdma-shared-dev-plugin-ds.yaml

```diff
@@ -53,6 +53,8 @@ spec:
             mountPath: /var/lib/kubelet/
           - name: config
             mountPath: /k8s-rdma-shared-dev-plugin
+          - name: rdma-devs
```

Comment:
so now the name is incorrect as the path now points to all devices :)  

maybe:
`name: devs`